modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2938,7 +2938,9 @@ definition {
 		Workflow.viewDefinitionUnpublished(workflowDefinitionTitle = "New Workflow Definition Title");
 	}
 
+	@description = "Verify that the user can save an invalid workflow definition as a draft"
 	@priority = 3
+	test CanSaveInvalidWorkflowDefinitionAsDraft {
```

Comment:
@davisaints I am thinking about this but maybe we can rename the test to not refer to it as draft, but as Not Published to make it consistent with the UI. I am also thinking that by reading this test, I cannot say that we are saving an invalid workflow definition and that it is unpublished after. Let me know what you think. Thanks!

Reply:
Make sense!